executable-spec/src/Cardano/Ledger/Spec/State/ProposalState.hs

```diff
@@ -10,7 +10,7 @@
 -- (improvement, implementation, etc).
 --
 module Cardano.Ledger.Spec.State.ProposalState
-  ( ProposalState (decision)
+  ( ProposalState (..)
```

Comment:
Oh, I see! There's a good reason why I didn't export these constructors :) We shouldn't use them. This module should be the sole responsible for invariants of the voting state.

Also I'd rather not export symbols using `..` since we might inadvertently export things that should remain private.